sites/fast-website/src/public/index.html

```diff
@@ -0,0 +1,120 @@
+              <fast-button slot="action" appearance="accent">Explore components</fast-button>
+            </site-section-header>
+            <fast-divider role="presentation"></fast-divider>
+            <fast-frame></fast-frame>
+          </div>
+        </section>
+
+        <section class="section" id="integration">
+          <div class="section-content">
+            <site-section-header>
+              <fast-badge slot="badge" class="section-badge">FULLY INTEGRATED</fast-badge>
+              <h2 class="section-heading">Works with existing frameworks</h2>
+              <p slot="body">Standards-based Web Components are the foundation of each FAST component, making them compatible with almost any modern web framework.</p>
+              <fast-button slot="action" appearance="accent">Learn More</fast-button>
+            </site-section-header>
+            <fast-divider role="presentation"></fast-divider>
+            <site-content-placement-container section="framework"></site-content-placement-container>
+          </div>
+        </section>
+
+        <section class="section" id="features">
+          <div class="section-content">
+            <site-section-header>
+              <fast-badge slot="badge" class="section-badge">A COMPREHENSIVE TOOLKIT</fast-badge>
+              <h2 class="section-heading">Features and utilities</h2>
+              <p slot="body">There are a million and one great ways to build your next website or application. To support the technologies you choose, creating unopinionated code is at the center of every decision we make in FAST.</p>
+              <fast-button slot="action" appearance="accent" href="/docs/en/contributing/install">Read documentation</fast-button>
+            </site-section-header>
+            <fast-divider role="presentation"></fast-divider>
+            <site-content-placement-container section="feature"></site-content-placement-container>
+          </div>
+        </section>
+
+        <section class="section community-section" id="community">
+          <div class="section-content">
+            <site-section-header>
+              <fast-badge slot="badge" class="section-badge">LEARN, MODIFY AND FOLLOW</fast-badge>
+              <h2 class="section-heading">Join the community</h2>
+              <p slot="body">Our community drives the rhythm and direction of FAST. By taking part, your voice will help shape our future.</p>
+            </site-section-header>
+            <fast-divider role="presentation"></fast-divider>
+            <site-content-placement-container section="community"></site-content-placement-container>
+          </div>
+        </section>
+
+          <footer class="footer">
+            <site-navigation>
+              <ul>
+                <li><fast-anchor appearance="lightweight" href="https://github.com/microsoft/fast-dna/blob/master/LICENSE">License</fast-anchor></li>
```

Comment:
If we plan on changing our GitHub name prior to launch we should pre-update the github URL before this goes in, no?

Reply:
I'm going to hold off on this update in my new branch until the change is made. This will redirect whereas the inverse will not be true until the change is made.